DS30.ipynb

```diff
@@ -149,8 +149,14 @@
     "def nlargest(n, word_scores):\n",
     "    return heapq.nlargest(n, word_scores, key=lambda x: x[1])\n",
     "\n",
+    "def remove_punctuation(text):\n",
+    "    # hardcoded to avoid importing any library\n",
+    "    exclude = '!\"#$%&\\'()*+,-./:;<=>?@[\\\\]^_`{|}~0123456789'\n",
```

Comment:
This is a little expansive.  I think numbers, `#` and `@` should definitely be kept (it is twitter after all).